### PR TITLE
fix: bug fix

### PR DESCRIPTION
**Pull request by DoctorVoid**

### CODE REVIEW

File: src/typeorm-bypasses/polaris-repository.ts [hunk 1]
```diff
@@ -16,6 +16,7 @@ import { PolarisCriteria } from '../contextable-options/polaris-criteria';
 import { PolarisFindManyOptions } from '../contextable-options/polaris-find-many-options';
 import { PolarisFindOneOptions } from '../contextable-options/polaris-find-one-options';
 import { PolarisSaveOptions } from '../contextable-options/polaris-save-options';
+import {CommonModel} from "..";
```
Comment (chenshoo) [inline src/typeorm-bypasses/polaris-repository.ts:19]: Why you havnt check in save?

Comment (DoctorVoid) [reply src/typeorm-bypasses/polaris-repository.ts:19]: Becuase its checked in the repository entity manger

Comment (DoctorVoid) [reply src/typeorm-bypasses/polaris-repository.ts:19]: > Why you havnt check in save?
Becuase its checked in the repository entity manger

File: src/typeorm-bypasses/polaris-entity-manager.ts [hunk 2]
```diff
@@ -146,7 +145,7 @@ export class PolarisEntityManager extends EntityManager {
         if (criteria instanceof PolarisFindManyOptions) {
             return super.find(
                 entityClass,
-                this.calculateCriteria<Entity>(entityClass, true, criteria),
+                this.findHandler.findConditions<Entity>(false, criteria),
```
Comment (chenshoo) [inline src/typeorm-bypasses/polaris-entity-manager.ts:148]: should be true

File: src/typeorm-bypasses/polaris-entity-manager.ts [hunk 3]
```diff
@@ -131,7 +130,7 @@ export class PolarisEntityManager extends EntityManager {
         if (criteria instanceof PolarisFindOneOptions) {
             return super.findOne(
                 entityClass,
-                this.calculateCriteria<Entity>(entityClass, true, criteria),
+                this.findHandler.findConditions<Entity>(false, criteria),
```
Comment (chenshoo) [inline src/typeorm-bypasses/polaris-entity-manager.ts:133]: should be true

File: src/typeorm-bypasses/polaris-entity-manager.ts (hunk 2 above)
Comment (DoctorVoid) [reply src/typeorm-bypasses/polaris-entity-manager.ts:148]: it was false all this time

**Pull request merged**